File: Linkpoint/src/main/java/com/linkpoint/slproto/objects/SLObjectPrimInfo.java
package com.linkpoint.slproto.objects;
import java.util.*;

import com.linkpoint.render.spatial.DrawListObjectEntry;
import com.linkpoint.render.spatial.DrawListPrimEntry;
import javax.annotation.Nonnull;

public class SLObjectPrimInfo extends SLObjectInfo {
    /* access modifiers changed from: protected */
    @Nonnull
    public DrawListObjectEntry createDrawListEntry() {
        return new DrawListPrimEntry(this);
    }

    public boolean isAvatar() {
        return false;
    }
}

File: Linkpoint/src/main/kotlin/com/linkpoint/slproto/objects/SLObjectPrimInfo.kt
package com.linkpoint.slproto.objects
import java.util.*

import com.linkpoint.render.spatial.DrawListObjectEntry
import com.linkpoint.render.spatial.DrawListPrimEntry
import javax.annotation.Nonnull

class SLObjectPrimInfo : SLObjectInfo() {
    /* access modifiers changed from: protected */
     public fun createDrawListEntry(): DrawListObjectEntry {
        return DrawListPrimEntry(this)
    }

     public fun isAvatar(): Boolean {
        return false
    }
}
